woocommerce/woocommerce-gutenberg-products-block
# Export `Button` and `Label` from `wc-blocks-checkout` package

This PR will revert #4222, but at the same time will make the `Button` and `Label` components available in the `wc-blocks-checkout` package.

By leaving `Button` and `Label` where they are and exporting them from the checkout package, we can allow existing blocks to import from `@woocommerce/base-components` components while extensions can use `@woocommerce/blocks-checkout`.


Partly addresses #4405




### How to test the changes in this Pull Request:

1. Check out 430-gh-woocommerce/woocommerce-points-and-rewards and run `npm run build`. Ensure no build errors happen.
2. Ensure the Points and Rewards plugin is activated.
3. Set up your user account so it has points available. (WooCommerce -> Points and Rewards)
3. Add items to your cart.
4. Go to the Cart/Checkout blocks and ensure the Points and Rewards sidebar is showing properly, it should look _similar_ to this: 
<img src="https://user-images.githubusercontent.com/5656702/123649886-04720100-d822-11eb-8e59-de186e07b1f4.png" width=300 />
